lib/Themes/OverrideDefaultTheme.php

```diff
@@ -19,6 +19,7 @@
 use OCP\IRequest;
 use OCP\IURLGenerator;
 use OCP\IUserSession;
+use OCA\NcTheming\Themes\CustomCss;
```

Comment:
You don't need the `use` for the file here since it's in the same namespace (this seems to be the error in the build pipeline, you showed me).